packages/forklift-console-plugin/src/modules/Providers/views/details/components/CredentialsSection/components/edit/OvirtCredentialsEdit.tsx

```diff
@@ -29,12 +33,58 @@ export const OvirtCredentialsEdit: React.FC<EditComponentProps> = ({ secret, onC
   const insecureSkipVerify = safeBase64Decode(secret?.data?.insecureSkipVerify || '') === 'true';
   const cacert = safeBase64Decode(secret?.data?.cacert || '');
 
+  const usernameHelperTextMsgs = {
+    error: t(
+      "Error: The format of the provided user name is invalid. Ensure the user name doesn't include whitespace characters.",
+    ),
+    warning: t(
+      'Warning: The provided user name does not include the user domain. Ensure the user name is in the format of username@user-domain. For example: admin@internal.',
+    ),
+    success: t(
+      'A user name for connecting to the Red Hat Virtualization Manager (RHVM) API endpoint. Ensure the user name is in the format of username@user-domain. For example: admin@internal.',
+    ),
+  };
+
+  const passwordHelperTextMsgs = {
+    error: t(
+      "Error: The format of the provided user password is invalid. Ensure the user password doesn't include whitespace characters.",
+    ),
+    success: t(
+      'A user password for connecting to the Red Hat Virtualization Manager (RHVM) API endpoint.',
+    ),
+  };
+
+  const insecureSkipVerifyHelperTextMsgs = {
+    error: t('Error: this field must be set to a boolean value.'),
+    success: t("If true (check box is checked), the provider's CA certificate won't be validated."),
+    hint: t(
```

Comment:
the hint is not a validation help message, e.g. success, warninig, error, it's a hint that is always the same, regardless of the validation state.
please create a new const for it.

Reply:
The const name `insecureSkipVerifyHelperTextMsgs` doesn't include the word 'verify' so I see it as a collection of text messages for this field and this specific hint is relevant for the `insecureSkipVerify` field only (explaining the enable/disable values).

But if you think that separating the help text based on component appearance is better then it's ok as well.

Reply:
yes, the idea is that you can do `insecureSkipVerifyHelperTextMsgs[ callVerificationMethod(text) ]` in cases it makes sense, do the keys of the text array should be the same as the verification options.